charts/trino/templates/service.yaml

```diff
@@ -14,6 +14,12 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+    {{- if .Values.server.config.https.enabled }}
```

Comment:
We have to think about existing deployments, where the `service.port` is already set to the same port as https - won't it cause issues to configure two same ports? This change also brings inconsistency - why one port is set using `service.port`, and another one using `server.config.https.port`?

Additionally, since there's no way to disable http, changing `service.port` is currently the only way of limiting access to it.

Reply:
I think there is already some deployment using a port for https, the administrator should choose another port for http, in the same case for http. I think this option is less problematic than having to manually release the port for https (source from several questions on the forum)

For example, I use a CD tool and I don't have access to kubectl because I'm part of the data team and not the devops team and having to patch the service in kubectl wasn't a very suitable solution for my scenario, which I imagine to be a common scenario and where we set up a hook so we can use it without manual intervention for each deployment.

As for the configuration, the place where I got the https port accessible was without values, if there is another place, we can improve this PR.

Reply:
Sorry, I don't understand how this responds to the concerns I raised.

Can you add a test for the scenario I described?